### ISSUE TITLE
Create installer packages for main distributions

### ISSUE THREAD

**Issue by andresgongora** [andresgongora/synth-shell#37]
- [ ] AUR
- [ ] DEB

Once #9 is done, create a AUR package

**Comment by olmari** [comment]
Also, DEB, then we cater to plenty usebase.

**Comment by andresgongora** [comment]
Good idea!! I'm creating a new issue for this in #43

**Comment by andresgongora** [comment]
Working on AUR package. This might take some time because it depends on #161 - I first want to divide synth-shell into sub-modules, then create an installer for them individually, and finally pack them all together.

**Comment by andresgongora** [comment]
not relevant since #161
